openAPI/quran.com/v4.json

```diff
@@ -2113,12 +2113,12 @@
         "tags": [
           "Quran"
         ],
-        "description": "Get a single #model:XCinMTfQdCrC3D3vH of all ayah.\n\nSee #endpoint:N4JAF8phDshhyrBHs endpoint to fetch available translations.\n\nYou can also include more fields of translation using `fields` query string.",
+        "description": "Get a single translation of the Qur'an\n\nUse https://api-docs.quran.com/docs/quran.com_versioned/translations to get the ID a desired translation",
```

Comment:
Another comments is that throughout the docs, we should use relative path and not full path and let Docusaurus resolve the link by itself which would be relative to the current directory for example, `https://api-docs.quran.com/docs/quran.com_versioned/translations` should be converted into `[link](./translations)` which would still be resolved to whatever directory and domain the current docs portal is one. This is more sustainable for the future.

Reply:
Not finished 100%